shared/api/storage_pool_volume.go

```diff
@@ -30,6 +33,9 @@ type StorageVolumePost struct {
 
 	// API extension: storage_api_remote_volume_snapshots
 	VolumeOnly bool `json:"volume_only" yaml:"volume_only"`
+
+	// API extension: custom_block_volumes
+	ContentType string `json:"content_type" yaml:"content_type"`
```

Comment:
Why is this one needed? If renaming, we don't need to know the type and if migrating, the migration protocol will tell us the type.

Reply:
Dropped